src/git.ts

```diff
@@ -23,6 +23,22 @@ export async function init(action: ActionInterface): Promise<void | Error> {
       action.silent
     )
 
+    await execute(`git remote rm origin`, action.workspace, action.silent)
+
+    try {
+      if (action.isTest === TestFlag.HAS_CHANGED_FILES) {
+        throw new Error()
+      }
+    } catch {
+      info('Attempted to remove origin but failed, continuing…')
+    }
```

Comment:
Is this actually doing something? Or did you want to put the execute into the try-catch block, too?

If so, I'd hand an extra flag to `isTest` to fake the error here?

Reply:
Ah yeah I just noticed that I didn't add the rm command in the try catch. I've updated the PR with an additional test flag for the test case too.